### ISSUE TITLE
Stop outputting long list of `Error parsing bundle asset` errors when the build fails

### ISSUE THREAD

**Issue by fregante** [webpack-contrib/webpack-bundle-analyzer#655]
When the build fails, webpack-bundle-analyzer fills up the logs with irrelevant lines:

```
Error parsing bundle asset "apps/admin-console/bundles/main-a005b5463f19ca11727d.min.js": no such file
Error parsing bundle asset "apps/admin-console/bundles/CustomFormComponent-648471e17196c5edc242.min.js": no such file
Error parsing bundle asset "apps/admin-console/bundles/WebAppPlatformLayout-3ff85759162b39d96eaa.bundle.min.js": no such file
Error parsing bundle asset "apps/admin-console/bundles/21983-1d2ef12ba1460dcedee5.bundle.min.js": no such file
Error parsing bundle asset "apps/admin-console/bundles/AdminConsoleLayout-86f8bbbfa0d21a23d0fe.bundle.min.js": no such file
Webpack Bundle Analyzer saved report to apps/admin-console/report.html
```

This time 100 because my build is generating several small bundles (icons)

The plugin should detect failure and avoid trying to read bundles that don't exist

**Comment by alexander-akait** [comment]
Do you use the latest version? Because this error should be fixed

**Comment by fregante** [comment]
```
    "webpack": "^5.96.1",
    "webpack-build-notifier": "^3.1.0",
    "webpack-bundle-analyzer": "^4.10.2",
```

**Comment by fregante** [comment]
I wonder if it's the same issue reported here https://github.com/webpack-contrib/webpack-bundle-analyzer/discussions/647

**Comment by valscion** [comment]
Yeah I haven't yet got to releasing a new version with https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/649 changes in. I was meaning to analyze the regression I saw in CI when I fixed it to again work before I'd release a new major version. I commented on https://github.com/webpack-contrib/webpack-bundle-analyzer/pull/650 for those mysterious cases that had now been failing but I hadn't yet figured out which changes could've caused the tests to start failing.

So it could be that once a new version with #649 are in, this would be fixed.

Can you test that out by following the "Develop with your own project" steps in CONTRIBUTING.md to see if your problem would be fixed with the changes we have in `master` but haven't yet released? https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/master/CONTRIBUTING.md#develop-with-your-own-project

**Comment by fregante** [comment]
Tried, not fixed

**Comment by alexander-akait** [comment]
@fregante Can you create reproducible test repo? Because it means your code is not parsable by acorn, that is very weird

**Comment by fregante** [comment]
> Because it means your code is not parsable by acorn, that is very weird

Yes, it happens, that's the issue. The real issue (syntax error) is lost in webpack-bundle-analyzer's output. What I do to repro it is just break a file by adding `/` at a random place. If that doesn't do it for you, I can try creating a repro later

**Comment by alexander-akait** [comment]
@fregante I fully understand why and when parsers are failed (and reasons), the question - why you have broken JS code in your output? And I ask about reproducible test repo to understand - **why you have broken code**.

**Comment by alexander-akait** [comment]
@fregante Got it, so we need to skip files which contains errors or was not generated

**Comment by fregante** [comment]
Correct.

### Repro

It's as basic as can be:

```sh
npm init -y
npm install webpack webpack-cli webpack-bundle-analyzer --save-dev
echo '/' > index.js
```

then copy-paste this into webpack.config.js

```json
const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;

module.exports = {
  plugins: [
    new BundleAnalyzerPlugin({analyzerMode: 'static', openAnalyzer:false})
  ]
}
```

and run

```sh
npx webpack ./index.js --mode production
```

## Output

```
Error parsing bundle asset "./main.js": no such file

No bundles were parsed. Analyzer will show only original module sizes from stats file.

Webpack Bundle Analyzer saved report to ./report.html
assets by status 261 bytes [cached] 1 asset
./index.js 2 bytes [built] [code generated] [1 error]

ERROR in ./index.js 1:1
Module parse failed: Unterminated regular expression (1:1)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> /
| 

webpack 5.98.0 compiled with 1 error in 87 ms
```

**Comment by valscion** [comment]
Thanks! I'll convert this to a feature request discussion to have them be in the same place ☺
